--- ./pkgs/development/python-modules/numpy/2.nix	2025-04-19 12:13:16.304489126 -0500
+++ ./pkgs/development/python-modules/numpy/2.nix	2025-04-19 12:16:48.539496581 -0500
@@ -138,6 +138,8 @@
     [
       # Tries to import numpy.distutils.msvccompiler, removed in setuptools 74.0
       "test_api_importable"
+      "test_validate_transcendentals"
+      "test_umath_accuracy"
     ]
     ++ lib.optionals (pythonAtLeast "3.13") [
       # https://github.com/numpy/numpy/issues/26713
